web/src/app/shared/services/workbasket/workbasket.service.ts

```diff
@@ -47,7 +47,7 @@ export class WorkbasketService {
 
     return this.domainService.getSelectedDomain()
       .pipe(mergeMap(domain => {
-        this.workbasketSummaryRef = this.httpClient.get<WorkbasketSummaryResource>(
+        this.workbasketSummaryRef = this.httpClient.get<WorkbasketSummaryRepresentation>(
           `${environment.taskanaRestUrl}/v1/workbaskets/${TaskanaQueryParameters
             .getQueryParameters(this.workbasketParameters(sortBy, order, name, nameLike, descLike, owner, ownerLike,
```

Comment:
Maybe even get this to use Filter and Sorting

Reply:
see above